hands-on-exercise/2-ignite-cli-adv/9-run-prod-docker.md

```diff
@@ -602,14 +609,20 @@ Not to forget, you inform Alice's validator that it should indeed listen on port
       -Ei 's/priv_validator_laddr = ""/priv_validator_laddr = "tcp:\/\/0.0.0.0:26659"/g' \
       /root/.checkers/config/config.toml
     ```
+
+  <HighlightBox type="note">
+
+  Make it listen on an IP address that is within the KMS private network.
+
+  </HighlightBox>
```

Comment:
I guess the identical highlight box can be taken out of the CodeGroup, and put at line 622

Reply:
Please do it.

Reply:
Done and pushed.